client/src/components/App.js

```diff
@@ -1,55 +1,34 @@
 import React, { Component } from 'react';
 import NavBar from './NavBar';
+import ChartJs from './ChartJs';
```

Comment:
Is both ChartJs and ReactCharts files needed? If not can you remove one and keep one please.

Reply:
I would use ChartJs because chart.js library is much customizable then react-charts.